docs-site/content/0.19.0/guide/features/filtering.md

```diff
@@ -1 +1,104 @@
 # Filtering
+
+Filtering is the process of excluding results that don't meet certain criteria. Filtering can help users narrow down their searches and find the most relevant result. For example, in a book dataset, users might be looking for a specific book by an author. They can filter the data with the author's name and limit the number of results they need to look through.
+
```

Comment:
To drive home the point of filtering vs faceting, could you add a note here along with a screenshot  of a widget from books-search.typesense.org saying something along the lines of:

To group and display the counts and values in this widget, we set this field as a facet. But once you click on a value in the widget, then a filter gets applied.